src/activities/rishabh/rishabh.js

```diff
@@ -0,0 +1,54 @@
+/* GCompris - rishabh.js
```

Comment:
I think rishabh.js file is not required in this activity. you can remove the unused files.